False positive on 1-tuple assignments containing a function call with a kwarg.

In v0.4.0 this code is flagged as a violation which I think is a false positive.

```
def some_func(x, y):
    pass


thing = (
    some_func(
        'a',
        y='b',
    ),
)
```

`example.py:8:11: T801 one element tuple`


However, neither of these variations produce an error:
```
def some_func(x, y):
    pass


thing = (
    some_func(
        'a',
        y='b'
    ),
)
```

```
def some_func(x, y):
    pass


thing = (
    some_func(
        'a',
        'b',
    ),
)
```

--- a/flake8_tuple.py
+++ b/flake8_tuple.py
@@ -126,7 +126,7 @@ def check_for_wrong_tuple(tree, code, noqa):
                 number_nl += 1
             if x.type == token.NEWLINE and ending_of_bad_tuple(previous_token):
                 errors.append(x.start)
-            if x.type == token.OP and x.string == '=':
+            if x.type == token.OP and x.string == '=' and previous_token.type != token.NAME:
                 x = TokenInfo(*next(tokens))
                 if x.type != token.OP and x.string != '(':
                     x_next = TokenInfo(*next(tokens))

--- a/tests/test_pep8_tuple.py
+++ b/tests/test_pep8_tuple.py
@@ -30,6 +30,7 @@ class Testflake8Tuple(unittest.TestCase):
         ("base_url = reverse(\n'test',\nargs=(pk,)\n)", 0),
         ("base_url = reverse(\n'test',\nargs=pk,\n)", 0),
         ("group_by = function_call('arg'),", 1),
+        ("group_by = (function_call(True, a=None, b=None),)", 0),
         ("group_by = ('foobar' * 3),", 1),
         ("val = {}.get(1),", 1),
         ("val = {}.get(\n1),", 1),

